pkg/services/alertmanager.go

```diff
@@ -33,13 +33,17 @@ type AlertmanagerNotification struct {
 
 // AlertmanagerOptions cluster configuration
 type AlertmanagerOptions struct {
-	Targets            []string   `json:"targets"`
-	Scheme             string     `json:"scheme"`
-	APIPath            string     `json:"apiPath"`
-	BasicAuth          *BasicAuth `json:"basicAuth"`
-	BearerToken        string     `json:"bearerToken"`
-	InsecureSkipVerify bool       `json:"insecureSkipVerify"`
-	Timeout            int        `json:"timeout"`
+	Targets             []string      `json:"targets"`
+	Scheme              string        `json:"scheme"`
+	APIPath             string        `json:"apiPath"`
+	BasicAuth           *BasicAuth    `json:"basicAuth"`
+	BearerToken         string        `json:"bearerToken"`
+	Timeout             int           `json:"timeout"`
+	InsecureSkipVerify  bool          `json:"insecureSkipVerify"`
+	MaxIdleConns        int           `json:"maxIdleConns"`
+	MaxIdleConnsPerHost int           `json:"maxIdleConnsPerHost"`
+	MaxConnsPerHost     int           `json:"maxConnsPerHost"`
+	IdleConnTimeout     time.Duration `json:"idleConnTimeout"`
```

Comment:
We can't use time.Duration for parsing time values - we will need to use a string and then parse the actual `time.Duration` by doing something like this 

```golang
	idleConnTimeout, err := time.ParseDuration(s.opts.IdleConnTimeout)
	if err != nil {
		return errors.Wrap(err, "failed to parse idle connection timeout")
	}
```

and then passing in `idleConnTimeout` to the `NewTransport` function.